pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md

```diff
@@ -23,7 +23,9 @@ La transferencia de un dominio permite cambiar el agente registrador que lo gest
 
 > [!warning]
 >
-> Si el dominio que está siendo modificado se encuentra registrado en OVHcloud, la transferencia entrante de dominio no es el procedimiento adecuado. Este procedimiento solo se aplica al cambio de dominio registrado (OVHcloud).
+> El *agente registrador* de un dominio representa la organización o proveedor acreditado en la que el dominio está registrado/suscrito por un particular, una asociación o una organización. Renovar la suscripción de su dominio (generalmente una vez al año) es a través del mismo agente registrador*.
```

Comment:
Maybe we should keep the "Registrar" termn in all translations ?

Reply:
done